pkg/processor/trigger/http/cors/cors.go

```diff
@@ -119,3 +131,18 @@ func (c *CORS) EncodePreflightMaxAgeSeconds() string {
 	}
 	return c.preflightMaxAgeSecondsStr
 }
+
+func (c *CORS) resolveAllowAllOrigins() bool {
+	if c.allowAllOrigins == nil {
+		false := false
+		c.allowAllOrigins = &false
+		for _, allowOrigin := range c.AllowOrigins {
+			if allowOrigin == "*" {
+				true := true
+				c.allowAllOrigins = &true
+				break
+			}
+		}
+	}
+	return *c.allowAllOrigins
```

Comment:
This is a mix of `resolveAllowAllOrigins()` (populating the field) and getting the actual value from the field `c.allowAllOrigins`.
It's only used once and directly so I think `c.allowAllOrigins` is needed at all this can be simplified greatly just returning the bool you need